examples/mcts_example.py

```diff
@@ -0,0 +1,396 @@
+
+if __name__ == "__main__":
+    # import botbowl.web.server as server
+    # server.start_server(debug=True, use_reloader=False, port=1234)
+
+    for i in [1, 3, 5, 11]:
+        print(f"-- Testing env {i} --")
+        # Load configurations, rules, arena and teams
+        config = botbowl.load_config(f"gym-{i}")
+        ruleset = botbowl.load_rule_set(config.ruleset)
+        arena = botbowl.load_arena(config.arena)
+        mcts_wins_env = 0
+        random_wins_env = 0
+        mcts_tds_env = 0
+        random_tds_env = 0
+        for as_home in [True, False]:
+            print("Playing as Home")
+            mcts_wins = 0
+            mcts_tds = 0
+            random_wins = 0
+            random_tds = 0
+            for _ in range(1):
+                home = botbowl.load_team_by_filename("human", ruleset, board_size=i)
+                away = botbowl.load_team_by_filename("human", ruleset, board_size=i)
+                config.competition_mode = False
+                config.debug_mode = False
+                config.fast_mode = True
+                config.pathfinding_enabled = True
+
+                # Play a game
+                bot_a = botbowl.make_bot("mcts")
+                bot_b = botbowl.make_bot("random")
+                home_bot = bot_a if as_home else bot_b
+                away_bot = bot_b if as_home else bot_a
+                game = botbowl.Game(1, home, away, home_bot, away_bot, config, arena=arena, ruleset=ruleset)
+                print("Starting game")
+                game.init()
+                print(f"{game.home_agent.name} score: {game.state.home_team.state.score}")
+                print(f"{game.away_agent.name} score: {game.state.away_team.state.score}")
+                visits, min_depths, max_depths, avg_depths = np.mean(bot_a.stats, axis=0)
+                print(f"MCTS Visits: {visits}, Min Depth: {min_depths}, Max. Depth: {max_depths}, Avg. Depth: {avg_depths}")
+                if game.get_winner() == home_bot and as_home:
+                    mcts_wins += 1
+                elif game.get_winner() == away_bot and not as_home:
+                    mcts_wins += 1
+                elif game.get_winner() is not None:
+                    random_wins += 1
+                if as_home:
+                    mcts_tds += game.state.home_team.state.score
+                    random_tds += game.state.away_team.state.score
+                else:
+                    random_tds += game.state.home_team.state.score
+                    mcts_tds += game.state.away_team.state.score
+            print("--------------------------")
+            print(f"MCTS wins: {mcts_wins}")
+            print(f"Random wins: {random_wins}")
+            print(f"MCTS TDs: {mcts_tds}")
+            print(f"Random TDs: {random_tds}")
+            mcts_wins_env += mcts_wins
+            random_wins_env += random_wins
+            mcts_tds_env += mcts_tds
+            random_tds_env += random_tds
+            break
```

Comment:
I think this `break` statement can be removed.

Reply:
Done